modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts

```diff
@@ -211,6 +210,7 @@ export class EddsaUtils extends baseTSSUtils<KeyShare> {
       publicShare: backupToBitgoPublicShare,
       privateShare: backupToBitgoPrivateShare,
       privateShareProof: await createShareProof(userGpgKey.privateKey, backupToBitgoPrivateShare.slice(0, 64)),
+      v: backupKeyShare.yShares[3].v,
```

Comment:
thoughts on calling this `vssProof`? we use more verbose/descriptive names for other props here